static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -55,6 +55,11 @@ const ErrorInfoChecks: Array<ErrorInfo> = [
     projectCheck: true,
     errorHelpType: ErrorHelpType.DYNAMIC_SERVER_USAGE,
   },
+  {
+    errorTitle: 'Hydration Error',
```

Comment:
title doesn't really work here, will need to look at messages

Reply:
@scttcper ok so you have to update this PR so it also looks at the message?

Reply:
i lied the title seems to contain it, just need to match the multiple options

Reply:
@scttcper easy to reuse the same `errorHelpType`